pkg/bwagreement/server_test.go

```diff
@@ -0,0 +1,154 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package bwagreement
+
+import (
+	"context"
+	"crypto"
+	"crypto/ecdsa"
+	"fmt"
+	"log"
+	"net"
+	"testing"
+
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+
+	"github.com/stretchr/testify/assert"
+
+	dbx "storj.io/storj/pkg/bwagreement/dbx"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/provider"
+)
+
+var (
+	ctx = context.Background()
+)
+
+const (
+	host     = "localhost"
+	port     = 5432
+	user     = "postgres"
+	password = "your-password"
+	dbname   = "pointerdb"
+)
+
+func getPSQLInfo() string {
+	psqlInfo := fmt.Sprintf("host=%s port=%d user=%s "+
+		"password=%s dbname=%s sslmode=disable",
+		host, port, user, password, dbname)
+	return psqlInfo
+}
+
+func TestBandwidthAgreements(t *testing.T) {
+	TS := NewTestServer(t)
+	defer TS.Stop()
+
+	signature := []byte("iamthedummysignatureoftypebyteslice")
+	data := []byte("iamthedummydataoftypebyteslice")
```

Comment:
Use realistic values here. Otherwise when we add verification, the tests will fail.